### ISSUE TITLE
Asterisk in `content` property generates warning about universal selector

### ISSUE THREAD

**Issue by awayken** [rossPatton/stylint#45]
When the `universal: true` flag is set, asterisks in content of a content property throw a false warning.

universal.styl
```stylus
input.required::after {
    content: "*";
}
```

universal.json
```json
{
    "alphabetical": true,
    "borderNone": true,
    "brackets": false,
    "colons": false,
    "colors": false,
    "commaSpace": true,
    "commentSpace": false,
    "cssLiteral": false,
    "depthLimit": false,
    "duplicates": true,
    "efficient": true,
    "enforceBlockStyle": false,
    "enforceVarStyle": false,
    "extendPref": false,
    "globalDupe": false,
    "indentSpaces": 4,
    "leadingZero": true,
    "maxWarnings": 10,
    "maxWarningsKill": false,
    "mixed": false,
    "namingConvention": false,
    "parenSpace": false,
    "placeholders": true,
    "semicolons": false,
    "universal": true,
    "valid": false,
    "whitespace": true,
    "zeroUnits": true,
    "zIndexDuplicates": false,
    "zIndexNormalize": false
}
```

Result:
```bash
Warning:  * selector is slow. Consider a different selector.
File: universal.styl
Line: 2: content: "*";
```

This warning occurs even if the asterisk is not the only content, for instance `content: "Required*"`.

**Comment by rossPatton** [comment]
should be an easy case to fix

**Comment by rossPatton** [comment]
pushing up a fix soon

**Comment by awayken** [comment]
Looks like the latest pull fixed this issue. Thanks!
